scripts/xtensa-build-zephyr.py

```diff
@@ -302,22 +283,60 @@ def check_west_installation():
 			"https://docs.zephyrproject.org/latest/getting_started/index.html")
 	print(f"Found west: {west_path}")
 
-def find_west_workspace():
-	"""[summary] Scans for west workspaces using west topdir command by checking script execution
-	west_top and SOF_TOP directories.
+def west_reinitialize(west_root_dir: pathlib.Path, west_manifest_path: pathlib.Path):
+	"""[summary] Performs west reinitialization to SOF manifest file asking user for permission.
+	Prints error message if script is running in non-interactive mode.
 
-	:return: [description] Returns path when west workspace had been found, otherwise None
-	:rtype: [type] bool, pathlib.Path
+	:param west_root_dir: directory where is initialized.
+	:type west_root_dir: pathlib.Path
+	:param west_manifest_path: manifest file to which west is initialized.
+	:type west_manifest_path: pathlib.Path
+	:raises RuntimeError: Raised when west is initialized to wrong manifest file
+	(not SOFs manifest) and script is running in non-interactive mode.
+	"""
+	global west_top
+	message = "West is initialized to manifest other than SOFs!\n"
+	message +=  f"Initialized to manifest: {west_manifest_path}." + "\n"
+	dot_west_directory  = pathlib.Path(west_root_dir.resolve(), ".west")
+	if args.no_interactive:
+		message += f"Try deleting {dot_west_directory } directory and rerun this script."
+		raise RuntimeError(message)
+	question = message + "Reinitialize west to SOF manifest? [Y/n] "
+	print(f"{question}")
+	while True:
+		reinitialize_answer = input().lower()
+		if reinitialize_answer == "y" or reinitialize_answer == "n":
+			break
+		sys.stdout.write('Please respond with \'Y\' or \'n\'.\n')
+
+	if not reinitialize_answer:
```

Comment:
I'm afraid this perform re-initialization even when the user answered 'n'?
```suggestion
	if reinitialize_answer != 'y':
```

Reply:
Thanks, was fixing in hurry and missed this one!